src/renderer/coremods/settings/pages/QuickCSS.tsx

```diff
@@ -22,6 +28,38 @@ type ThemeModule = {
   removeChangeListener: (listener: () => unknown) => unknown;
 };
 
+const PopoutModule = await webpack.waitForModule(
+  webpack.filters.bySource('type:"POPOUT_WINDOW_OPEN"'),
+);
+// eslint-disable-next-line @typescript-eslint/non-nullable-type-assertion-style
```

Comment:
All these comments to disable the rules are useless. Instead use the `!` assertion, or even better check later if it's undefined or not.